### PR TITLE
Problem: outdated README

### PR DESCRIPTION
**Pull request by calvinlauyh**
Solution: (Fix #80) Updated README with rebranded name

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ There are different tests that can be executed in the following ways:
 - [Project Website](http://chain.crypto.com/)
```
Comment (lezzokafka) [inline readme.md:123]:
```suggestion
- [Project Website](http://crypto.org/)
```

File: readme.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ There are different tests that can be executed in the following ways:
...
 - [Technical Documentation](http://chain.crypto.com/)
```
Comment (lezzokafka) [inline readme.md:124]:
```suggestion
- [Technical Documentation](http://crypto.org/docs)
```

File: LICENSE [hunk 3]
```diff
@@ -1,4 +1,4 @@
-  Copyright 2018 - 2020, Foris Limited ("Crypto.com"). 
+  Copyright 2021, CRO Protocol Labs ("Crypto.org").
```
Comment (tomtau) [inline LICENSE:1]: the original code was developed as per that copyright, so perhaps keep both (one 2018-2020, one 2021)?

File: SECURITY.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+3. Crypto.org will acknowledge the bug, and make the necessary code changes to patch it. Some issues may require more time than others to patch, but we will strive to patch each vulnerability as quickly as our resources and development process allow.
+4. Crypto.org will publicly release the security patch for the vulnerability, and acknowledge the security fix in the release notes once the issue has been resolved. Public release notes can reference to the person or people who reported the vulnerability, unless they wish to stay anonymous.
 
 ## Contact Us
-If you find a security issue, you can report it on the [Crypto.com HackerOne Bug Bounty Program](https://hackerone.com/crypto) or you can contact our team directly at [chain-security@crypto.com](mailto:chain-security@crypto.com).
+If you find a security issue, you can report it on the [Crypto.org HackerOne Bug Bounty Program](https://hackerone.com/crypto) or you can contact our team directly at [chain-security@crypto.com](mailto:chain-security@crypto.com).
```
Comment (tomtau) [inline SECURITY.md:17]: I think the security policy doesn't need to be renamed like this, given afaik there's no dedicated "crypto.org security team" or bounty program or something like that -- all that is handled by crypto.com?

**Review by tomtau — CHANGES_REQUESTED** [review]
looks ok, but not sure whether the renaming make sense in certain places

File: LICENSE (hunk 3 above)
Comment (calvinaco) [reply LICENSE:1]: Sure, will do

**Review by tomtau — APPROVED** [review]

**Pull request merged**